content/en/blog/_posts/2020-05-20-my-exciting-journey-into-kubernetes-history.md

```diff
@@ -0,0 +1,655 @@
+---
+title: "My exciting journey into Kubernetes’ history"
+date: 2020-05-20
```

Comment:
```suggestion
date: 2020-05-28
```

Reply:
Alright, changed the date in the files and directories, too :+1: